VocaDbWeb.Core/VocaDbWeb.Core.csproj

```diff
@@ -5,6 +5,8 @@
     <RootNamespace>VocaDb.Web</RootNamespace>
     <LangVersion>preview</LangVersion>
     <Nullable>enable</Nullable>
+    <!-- CA2300: Do not use insecure deserializer BinaryFormatter -->
+    <EnableUnsafeBinaryFormatterSerialization>true</EnableUnsafeBinaryFormatterSerialization>

```

Comment:
Can we enable unsafe binary serialization temporally? Or should we replace it with a safe one *now*?

Reply:
It's ok to have this for now, but maybe add to the comment mentioning where is it being used, and note that it should be removed when possible.